docs/pages/includes/sso/tctlconnection.mdx

```diff
@@ -0,0 +1,40 @@
+<Details 
```

Comment:
How would you describe the division of labor between this partial and `tctl.mdx`?